### PR TITLE
Add support for pronto.io

### PR DESCRIPTION
**Pull request by skylon07**
The main problem I noticed was one button had an incorrect background

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -44,6 +44,15 @@ div[class^='scrollableList'] {
 
 ================================
 
+*.pronto.io
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:47]:
```suggestion
pronto.io
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -44,6 +44,15 @@ div[class^='scrollableList'] {
...
+
+CSS
+input.btn-new-group.pointer {
+    background-color: rgba(0,0,0,0) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:51]:
```suggestion
    background-color: var(--darkreader-neutral-background) !important;
```

**Review by Gusted — CHANGES_REQUESTED** [review]

**Review by Gusted — APPROVED** [review]

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -44,6 +44,15 @@ div[class^='scrollableList'] {
 
 ================================
 
+pronto.io
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:47]: Please ensure that the site is now at the correct entry.

**Comment by skylon07** [comment]
Sorry, it still says "1 workflow awaiting approval"

**Pull request merged**